.travis.yml

```diff
@@ -24,14 +24,16 @@ before_install:
   - ./runisort
 
 install:
-  - pip install $DJANGO
```

Comment:
don't remove this, put back the build matrix, exclude python2.7 for django 2.0 (research how to do this)